### PR TITLE
Proto to grpc comments propagation

### PR DESCRIPTION
**Pull request by dsborets**
I'm proposing to use standard protobuf description set file as a source of comments in order to propagate comments from proto to GraphQL schema

The proto description set file could be generated by protobuf plugin:

Maven:
```
   <plugin>
       <groupId>org.xolstice.maven.plugins</groupId>
       <artifactId>protobuf-maven-plugin</artifactId>
       <version>0.5.1</version>
       <configuration>
         <protocExecutable>/usr/local/bin/protoc</protocExecutable>
         <writeDescriptorSet>true</writeDescriptorSet>
         <descriptorSetFileName>descriptor_set.desc</descriptorSetFileName>
         <descriptorSetOutputDirectory>${build.directory}/test-classes/META-INF/proto</descriptorSetOutputDirectory>
         <includeSourceInfoInDescriptorSet>true</includeSourceInfoInDescriptorSet>
         <includeDependenciesInDescriptorSet>true</includeDependenciesInDescriptorSet>
       </configuration>
       <executions>
         <execution>
           <goals>
             <goal>compile</goal>
             <goal>test-compile</goal>
           </goals>
         </execution>
       </executions>
    </plugin>
```

Grails:
```
protobuf {
    protoc { artifact = "com.google.protobuf:protoc:${protocVersion}" }
    plugins {
        grpc { artifact = "io.grpc:protoc-gen-grpc-java:${grpcVersion}" }
    }
    generateProtoTasks {
        ofSourceSet('main').each{task ->
            task.generateDescriptorSet = true
            task.descriptorSetOptions.includeImports = true
            task.descriptorSetOptions.includeSourceInfo = true
            task.descriptorSetOptions.path = "${buildDir}/resources/main/META-INF/proto/descriptor_set.desc"
            task.plugins {
                grpc {}
            }
        }
    }
}
```

The generated protofile description file must be named as `descriptor_set.desc` and located in `META-INF/proto` of the final jar file

### CODE REVIEW

File: rejoiner/pom.xml [hunk 1]
```diff
@@ -258,6 +258,11 @@
        <version>0.5.1</version>
```
Comment (leoconco) [inline rejoiner/pom.xml:258]: there's a 0.6.1 version, any reason to not use that?

File: rejoiner/src/main/java/com/google/api/graphql/rejoiner/SchemaModule.java [hunk 2]
```diff
@@ -97,26 +97,32 @@ protected void addMutationList(List<GraphQLFieldDefinition> mutations) {
...
+  protected void configureSchema() {
+  }
 
   @Override
   protected final void configure() {
     configureSchema();
     Multibinder<GraphQLFieldDefinition> queryMultibinder =
         Multibinder.newSetBinder(
-            binder(), new TypeLiteral<GraphQLFieldDefinition>() {}, Annotations.Queries.class);
+            binder(), new TypeLiteral<GraphQLFieldDefinition>() {
```
Comment (leoconco) [inline rejoiner/src/main/java/com/google/api/graphql/rejoiner/SchemaModule.java:108]: this is noise

File: rejoiner/src/main/java/com/google/api/graphql/rejoiner/SchemaModule.java [hunk 3]
```diff
@@ -97,26 +97,32 @@ protected void addMutationList(List<GraphQLFieldDefinition> mutations) {
     allMutationsInModule.addAll(mutations);
   }
 
-  protected void configureSchema() {}
+  protected void configureSchema() {
```
Comment (leoconco) [inline rejoiner/src/main/java/com/google/api/graphql/rejoiner/SchemaModule.java:100]: this is noise

File: rejoiner/src/main/java/com/google/api/graphql/rejoiner/SchemaModule.java [hunk 4]
```diff
@@ -97,26 +97,32 @@ protected void addMutationList(List<GraphQLFieldDefinition> mutations) {
...
     configureSchema();
     Multibinder<GraphQLFieldDefinition> queryMultibinder =
         Multibinder.newSetBinder(
-            binder(), new TypeLiteral<GraphQLFieldDefinition>() {}, Annotations.Queries.class);
+            binder(), new TypeLiteral<GraphQLFieldDefinition>() {
+            }, Annotations.Queries.class);
     Multibinder<GraphQLFieldDefinition> mutationMultibinder =
         Multibinder.newSetBinder(
-            binder(), new TypeLiteral<GraphQLFieldDefinition>() {}, Annotations.Mutations.class);
+            binder(), new TypeLiteral<GraphQLFieldDefinition>() {
```
Comment (leoconco) [inline rejoiner/src/main/java/com/google/api/graphql/rejoiner/SchemaModule.java:112]: this is noise ...
many changes are  formatting noise

**Review by leoconco — COMMENTED** [review]
Nice, cut some noise changes.
Are you proposing this to the google repo?

File: rejoiner/pom.xml (hunk 1 above)
Comment (dsborets) [reply rejoiner/pom.xml:258]: I don't want to update a lib version in my PR. Only what is really necessary for this particular functionality. Hopefully the maintainers will upgrade libs soon...

**Pull request merged**